### ISSUE TITLE
Host `*.deb` releases in GitHub releases-hosted APT repostories

### ISSUE THREAD

**Issue by rpatterson** [MediaBrowser/Emby#3283]
I've built a small [Python utility](https://github.com/rpatterson/github-apt-repos) that builds signed APT (IOW Debian/Ubuntu) repositories from *existing* `*.deb` releases such that users can add this special GitHub release as an APT repository so their `/etc/apt/sources.list.d` and be kept up to date.

I've set up sample APT repositories for [deb-amd64](https://github.com/rpatterson/Emby.Releases/releases/tag/apt-deb-amd64), [deb-armhf](https://github.com/rpatterson/Emby.Releases/releases/tag/apt-deb-armhf), and [netgear-amd64](https://github.com/rpatterson/Emby.Releases/releases/tag/apt-netgear-amd64).  I've only "dogfooded" these enough to ensure that ``emby-server`` installs from ``deb-amd64``.  I haven't actually used this to replace ``emby-server`` where I actually use it because that installation is from the outdated [build.opensuse.org](https://build.opensuse.org/package/requests/home:emby/emby-server) and I need to put some time in to migrate it to the new release runtime preserving exisitng data.

I'm available to adjust and/or improve `github-apt-repos` as needed to make it useful to this project (or others).  For context, here's [the forum thread that inspired this contribution](https://emby.media/community/index.php?/topic/58532-emby-3400-opensuse-repositories/).

**Comment by rpatterson** [comment]
I just used this to upgrade to the latest official Emby release and everything seems to be working fine.  IOW, I'm fully "dogfood"ing this solution now.

**Comment by coltc50** [comment]
Great!

Is there a way to make this work with the beta versions?

**Comment by rpatterson** [comment]
Good point, that's a better way for the tool to work, to specify the release URL and default to getting the latest release URL from a git checkout.  Working on that now.

**Comment by fbartels** [comment]
Hi @rpatterson, are you still maintaining your repository?
